PicaSWB.js

```diff
@@ -39,6 +48,16 @@ var nachnameMapping = {
 };
 var nameMapping = {
```

Comment:
Ich habe jetzt einmal einen ersten Versuch für den Lookup über das Web gemacht: https://github.com/UB-Mannheim/zotkat/commit/aaf7d9a37add8b411135802c02c9b14d1385e513

Leider funktioniert dies nur, wenn man die Einträge exportiert (Quick-Copy geht nicht). Darum ist dies momentan in einem speziellen Branch und ich hoffe, dass man dies noch besser machen kann.